components/Starknet/modules/tools/pages/limits_and_triggers.adoc

```diff
@@ -41,6 +44,22 @@ The gas limit for Starknet blocks is therefore inherited from the gas limit for
 The gas limit for Starknet blocks is inherited from the gas limit for Ethereum blocks.
 ====
 
+## Max transaction size (Cairo steps)
```

Comment:
Change all # headings to AsciiDoc = headings.

Reply:
Done